skimage/util/dtype.py

```diff
@@ -313,7 +313,7 @@ def _convert(image, dtype, force_copy=False, uniform=False):
         computation_type = _dtype_itemsize(itemsize_in, dtype_out,
                                            np.float32, np.float64)
 
-        if kind_in == 'u':
+        if kind_in == 'u' or kind_in == 'i':
```

Comment:
From how the code unit is written, the case of signed ints is handled by the `else` statement. A good way to express the changes you mean would be to update or add tests so they do exactly what you're looking for, then make the changes in the codebase so that the new tests pass (TDD, for test-driven development).

Reply:
Good point. It was handled by else statement, but that was one of the main problems in [#5281](https://github.com/scikit-image/scikit-image/pull/5281#issue-597330331) . I'll make change in unit test